.github/ISSUE_TEMPLATE/proposal.md

```diff
@@ -8,22 +8,24 @@ assignees: ''
 ---
 
 **Describe what you are trying to solve**
-A description of the current limitation/problem/challenge that you are experiencing.
+<!-- A description of the current limitation/problem/challenge that you are experiencing.
```

Comment:
where is the closing tag matching this one?

Reply:
my fault, fixed it.